include/zephyr/bluetooth/iso.h

```diff
@@ -172,77 +174,86 @@ struct bt_iso_chan {
 	 */
 	bt_security_t			required_sec_level;
 #endif /* CONFIG_BT_SMP && CONFIG_BT_ISO_UNICAST */
-	/** Node used internally by the stack */
+	/** @internal Node used internally by the stack */
 	sys_snode_t node;
 };
 
 /** @brief ISO Channel IO QoS structure. */
 struct bt_iso_chan_io_qos {
-	/** Channel SDU. Maximum value is BT_ISO_MAX_SDU */
+	/**
+	 * @brief Channel SDU.
+	 *
+	 * Value range is @ref BT_ISO_MIN_SDU to @ref BT_ISO_MAX_SDU.
+	 */
 	uint16_t			sdu;
-	/** Channel PHY - See BT_GAP_LE_PHY for values.
-	 *  Setting BT_GAP_LE_PHY_NONE is invalid.
+	/**
+	 * @brief Channel PHY - See BT_GAP_LE_PHY for values.
```

Comment:
Maybe just saying "See LE PHY types for values." is clearer as the enum is nameless and `BT_GAP_LE_PHY` doesn't refer to anything.